### DISCUSSION TITLE
Conditional Formatting of Row or Cell Based on Text in Column

### DISCUSSION THREAD

**Discussion by noahphex** [dfinke/ImportExcel#1655] [Q&A] [answered]
I've been struggling with the below two Add-ConditionalFormatting calls. Neither returns an error but when I open the Excel file it claims its corrupted and I'm not getting the formatting I need. 

This call needs to color any row LightBlue when the text in column N has "true" in it. 
Add-ConditionalFormatting -Worksheet $myworksheet -Range"A1:N400" -RuleType Expression -ConditionValue '=N1="TRUE"' -BackgroundColor LightBlue

And this one should highlight the cell in column F if the text in column G has "True" in it:
Add-ConditionalFormatting -Worksheet $myworksheet -Range "F2:F400" -RuleType Expression -ConditionValue '=G1="TRUE"' -BackgroundColor Yellow

I've searched high and low and banged my head against a wall trying to get this to work, unsuccessfully, so if anyone has any tips I'd appreciate it.

**Comment by noahphex** [comment, marked as answer]
One always figures it out, after they ask for help. I got it working.

Add-ConditionalFormatting -Worksheet $ws -Range "A1:N$lastrow" -RuleType Expression -ConditionValue '=$N1="TRUE"' -BackgroundColor LightBlue

Add-ConditionalFormatting -Worksheet $ws -Range "F:F" -RuleType Expression -ConditionValue '=$G1="TRUE"' -BackgroundColor Yellow

**Comment by dfinke** [comment]
"talking to the rubber duck" refers to a debugging technique known as rubber duck debugging
 
 Glad you got it sorted.
